.gitignore

```diff
@@ -1,2 +1,5 @@
-build/*
+build/*.o
 lib/*.o
+asciify_base
```

Comment:
Aren't they supposed to be in `build/` folder? Also, line 1 of `.gitignore` was correct, because all final binaries were in that directory, and now it just ignores `*.o`

Reply:
Also, it looks like object files are now in `build/` folder, so we don't need `lib/*.o` in `.gitignore`, I think

Reply:
Yes, sorry, i should've cleaned it.